Build/Projects/MonoGame.Framework.definition

```diff
@@ -35,14 +35,14 @@
     <PlatformSpecificOutputFolder>True</PlatformSpecificOutputFolder>
 
     <CustomDefinitions>
-      <Platform Name="Android">TRACE;ANDROID;GLES;OPENGL;OPENAL</Platform>
-      <Platform Name="iOS">IOS;GLES;OPENGL;OPENAL</Platform>
-      <Platform Name="tvOS">IOS;TVOS;GLES;OPENGL;OPENAL</Platform>
-      <Platform Name="Linux">TRACE;LINUX;OPENGL;OPENAL;DESKTOPGL;SUPPORTS_EFX</Platform>
-      <Platform Name="MacOS">MONOMAC;OPENGL;OPENAL;DESKTOPGL;SUPPORTS_EFX</Platform>
-      <Platform Name="WindowsGL">TRACE;WINDOWS;DESKTOPGL;SUPPORTS_EFX</Platform>
-      <Platform Name="Windows">TRACE;WINDOWS;DIRECTX;WINDOWS_MEDIA_SESSION</Platform>
-      <Platform Name="WindowsUniversal">TRACE;NETFX_CORE;WINDOWS_UAP;DIRECTX;DIRECTX11_1;WINDOWS_MEDIA_ENGINE</Platform>
+      <Platform Name="Android">TRACE;ANDROID;GLES</Platform>
```

Comment:
A bit of an opinion on this, but it makes sense for ie. Android to have `OPENGL` define, I wouldn't remove any defines, they might need to be used later and they aren't really hurting anybody.

Reply:
Agreed.  Common OpenGL code will be under the `OPENGL` define.  OpenGL ES-specific code will be under the `GLES` define.

Reply:
I did not remove the define. Android uses the OpenGLGraphics service which defines the OPENGL symbol.